back/engines/commercial/verification/lib/verification/feature_specification.rb

```diff
@@ -28,7 +28,7 @@ def self.feature_description
           {
             type: 'object',
             title: method.name,
-            required: ['name', *method.config_parameters],
+            required: ['name'],
```

Comment:
This makes all config fields from all verification methods optional. I thought that was better than adding an extra configuration for this. If you're setting this up, you should probably know what you're doing, or follow a guide that tells you what to fill out.